src/components/ShareModal.vue

```diff
@@ -63,6 +65,10 @@ export default {
                 alert(this.$t("info.cannot_copy"));
             }
         },
+        onChange() {
+            localStorage.setItem("shareWithTimeCode", this.withTimeCode);
+            localStorage.setItem("shareAsPipedLink", this.pipedLink);
```

Comment:
You should use the setPreference method, since this can fail if the user has cookies disabled.